### PR TITLE
Recommend fixes

### PR DESCRIPTION
**Pull request by lyriccoder**
Fixing 
https://github.com/yegor256/aibolit/issues/263
https://github.com/yegor256/aibolit/issues/264

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#266](https://github.com/yegor256/aibolit/issues/266) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/CT5UB7XAP/6b9f4637-a63d-4cfd-8ecd-f2c27f653e75); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @acheshkov[/z](https://www.0crat.com/u/acheshkov) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: aibolit/__main__.py [hunk 1]
```diff
@@ -200,19 +201,21 @@ def calculate_patterns_and_metrics(file):
     return input_params, code_lines_dict, error_string
 
 
-def inreference(input_params: List[int], code_lines_dict):
+def inreference(input_params: List[int], code_lines_dict, args):
```
Comment (acheshkov) [inline aibolit/__main__.py:204]: > `inreference`

is it typo ?

**Comment by lyriccoder** [comment]
@acheshkov fixed, tests passed

**Comment by acheshkov** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 18 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Job audit: performer @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)[/z](https://www.0crat.com/u/carlosmiranda) didn't make CR comments

**Pull request merged**